src/editor/editor.js

```diff
@@ -141,6 +143,26 @@ export class HeynoteEditor {
         })
     }
 
+    setBufferPath(path) {
+      if (this.loadFunction) {
+        this.loadFunction(path).then((content) => {
+          this.view.dispatch(
+            this.view.state.update(
```

Comment:
@heyman This state update is not working but I'm not sure why. The content is correctly read out, but the UI won't update the editor with the new contents. I'd appreciate it if you could take a look.